### DISCUSSION TITLE
ARA over AG

### DISCUSSION THREAD

**Discussion by abysalvinthomas** [apohl79/audiogridder#807] [Ideas]
This is my wild dream...
Is it possible ARA over AG..?

https://github.com/Celemony/ARA_SDK

**Comment by graywyot** [comment]
If possible ARA over AG, that would be a great feature.

**Reply by abysalvinthomas** [reply]
i have no technical know how ...
but i believe, it will be a leap forwarded step.


waiting for the responds of 
Andreas Pohl apohl79
